### PR TITLE
auto-generate releases page from version manifest files

### PR DESCRIPTION
**Pull request by akphi**
@epsstan Please make comment on the structure of the `manifest.json` file so I can adjust accordingly.

If you try this locally, it should bring up the website for releases at http://localhost:3000/releases
Also, in the doc, you should see a link on the top right corner with label `Releases`, and this is how we're supposed to reach it from the main doc.

### CODE REVIEW

**Comment by epsstan** [comment]
Apologies for the belated comments.

At the risk of duplication, it might be better to also have the version number in the manifest.

I think we decided to punt on the location of the release notes till we make the next release. In any case, in releases.js we can derive the location of the release notes instead of explicitly adding the release notes url to the manifest.

**Comment by epsstan** [comment]
I'm generally not a fan of the top right nav bar. When you first navigate to the home page you see "Github | Docs". And once you click on "Docs" you lose the top right nav bar. 

There is now a similar inconsistency with the left nav bar. Once you navigate to "releases", you lose the left nav bar.

File: website/pages/releases.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            <thead>
+              <tr>
+                <th>Version</th>
+                <th>
+                  <a href="https://hub.docker.com/r/finos/legend-studio">
+                    Legend Studio
+                  </a>
+                </th>
+                <th>
+                  <a href="https://hub.docker.com/r/finos/legend-engine">
```
Comment (epsstan) [inline website/pages/releases.js:57]: url is finos/legend-engine-server. Same with sdlc

Comment (akphi) [reply website/pages/releases.js:57]: Will fix!

**Comment by akphi** [comment]
> Apologies for the belated comments.
> 
> At the risk of duplication, it might be better to also have the version number in the manifest.
> 
> I think we decided to punt on the location of the release notes till we make the next release. In any case, in releases.js we can derive the location of the release notes instead of explicitly adding the release notes url to the manifest.

Done!


> I'm generally not a fan of the top right nav bar. When you first navigate to the home page you see "Github | Docs". And once you click on "Docs" you lose the top right nav bar.
> 
> There is now a similar inconsistency with the left nav bar. Once you navigate to "releases", you lose the left nav bar.

This will be much easier to achieve when we upgrade to `Docusaurus@2` so let's do it then

**Review by epsstan — APPROVED** [review]

**Pull request merged**